packages/vite/src/addons.ts

```diff
@@ -0,0 +1,23 @@
+import { ResolverLoader, packageName } from '@embroider/core';
```

Comment:
added this back because i was getting duplicate packages, like glimmer-vm which does not like it